### PR TITLE
Set user agent for Logging client

### PR DESCRIPTION
**Pull request by mziccard**

### CODE REVIEW

**Comment by mziccard** [comment]
/cc @lesv 

Codacy seems to be stuck on this (even though it reports failure).

**Comment by mziccard** [comment]
Update: Codacy is now done and happy

File: gcloud-java-logging/src/main/java/com/google/cloud/logging/spi/DefaultLoggingRpc.java [hunk 1]
```diff
@@ -97,13 +99,18 @@ private InternalLoggingOptions(LoggingOptions options) {
   public DefaultLoggingRpc(LoggingOptions options) throws IOException {
     executorFactory = new InternalLoggingOptions(options).executorFactory();
     executor = executorFactory.get();
+    String libraryName = options.libraryName();
```
Comment (lesv) [inline gcloud-java-logging/src/main/java/com/google/cloud/logging/spi/DefaultLoggingRpc.java]: Always gcloud-java from what I can see.  I guess that's a good thing.

**Comment by lesv** [comment]
LGTM - I presume I'll be seeing this kind of CL for all rpc's.

**Comment by mziccard** [comment]
> LGTM - I presume I'll be seeing this kind of CL for all rpc's.

User agent is all set for other rpc's. Just logging was left behind.

**Pull request merged**